cn.navclub.vdbfx.json

```diff
@@ -0,0 +1,84 @@
+{
+    "app-id": "cn.navclub.dbfx",
+    "runtime": "org.gnome.Sdk",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "v-dbfx",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "libgee",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/GNOME/libgee.git",
+                    "commit": "4ed45f6f32dcb8712501d38a44ec754d4937a26e"
+                }
+            ]
+        },
+        {
+            "name": "sqlite",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./configure --prefix=/app",
+                "make",
+                "make install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.sqlite.org/snapshot/sqlite-snapshot-202205171511.tar.gz",
+                    "sha1": "90d18bd735f96111df61c4dc78c1f011ac95ad1b"
+                }
+            ]
+        },
+        {
+            "name": "flatpak_mysql",
+            "buildsystem": "cmake-ninja",
+            "cleanup": [
+                "lib",
+                "include"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "branch": "main",
+                    "url": "https://github.com/GZYangKui/flatpak-mysql.git",
+                    "commit": "a4d48c6323d4afa3ecf5b999ccab52e55d963881"
+                }
+            ]
+        },
+        {
+            "name": "dbfx",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "branch": "master",
+                    "url": "https://github.com/databasefx/v-dbfx.git",
+                    "commit": "28816a7d807a69f2479ba1d424668c54ad2e574a"
```

Comment:
Where's the source? https://github.com/databasefx/v-dbfx/tree/28816a7d807a69f2479ba1d424668c54ad2e574a

Reply:
This software is written by me and belongs to closed source software. I don't know how to fill in here

Reply:
yeah I was look at this and has the same question.

So that mean the `MIT` license in the appdata is a lie?

Reply:
If it is closed source software, what should be filled in the open source agreement?

Reply:
You set the buildsystem as meson in the module, but the repo has nothing but text files and images. Are you building locally? Then provide a tarball with binaries.